DevIncubator-Autopark/Program.cs

```diff
@@ -1 +1,74 @@
-﻿Console.WriteLine("Hello, World!");
+﻿namespace DevIncubatorAutopark
+{
+    class Program
+    {
+        public static void Main()
+        {
+            //1
+            VehicleType[] vehicleTypes = new VehicleType[]
+            {
+               new VehicleType("Bus", 1.2f),
+               new VehicleType("Car"),
+               new VehicleType("Rink", 1.5f),
+               new VehicleType("Tractor", 1.2f)
+            };
+
+            //2
+            foreach (var vehicleType in vehicleTypes)
+            {
+                vehicleType.Display();
+            }
+
+            //3
+            vehicleTypes.Last().TaxCoefficient = 1.3f;
```

Comment:
At this project, you can't use Linq.
You can write vehicleTypes[^1]

Reply:
fixed